build/scripts/tests.sh

```diff
@@ -1,130 +0,0 @@
-#!/usr/bin/env bash
-# Copyright (c) .NET Foundation and contributors. All rights reserved.
-# Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-set -e
-set -u
-
-if [[ "${1}" =~ ^(-h|-help|--help|-\?|/\?)$ ]]; then
-    echo "USAGE:"
-    echo "build/scripts/tests.sh (Debug|Release) (dotnet|mono|mono-debug) [test assembly name] [xunit args]"
-    echo "If specified, the test assembly name must be a substring match for one or more test assemblies."
-    echo "Note that it's a substring match so '.dll' would match all unit test DLLs and run them all."
-    echo "Any xunit args specified after the assembly name will be passed directly to the test runner so you can run individual tests, i.e. -method \"*.Query_01\""
```

Comment:
This script had two features that aren't immediately obvious how to do on the new system:

1. Run a given test project under Mono
2. Run an individual test project vs. the entire set

Reply:
1. `msbuild /t:Test /t:TestRuntime=Mono MyProj.csproj`
2. `msbuild /t:Test MyProj.csproj`

Reply:
Also:
`msbuild /t:Test /t:TestRuntime=Mono /t:MonoTool="mono --debug" MyProj.csproj` to pass parameters or choose where `mono` is executed from.